packages/legend-data-cube/src/stores/core/__tests__/DatacubeQuerySnapshotBuilderTestUtils.ts

```diff
@@ -24,6 +24,17 @@ export type DataCubeQuerySnapshotBuilderTestCase = [
   string | undefined, // error
 ];
 
+export type OperationSnapshotAnalysisTestCase = [
```

Comment:
Let's prefix them with `DataCube...` and we can combine them, if 3rd param is provided, we do the check differently.